### PR TITLE
Move top-level affiliations to index document root

### PR DESCRIPTION
**Pull request by brinxmat**
- Moves the top-level affiliations to index document root
- Unfortunately includes blank-node ids on maps

### CODE REVIEW

File: expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java [hunk 1]
```diff
@@ -277,6 +253,41 @@ void shouldNotFailWhenInputContainsAffiliationsThatAreIncomplete() {
...
+            .flatMap(ExpandedResourceTest::getaVoid)
+            .forEach(ExpandedResourceTest::assertAllFieldsAreValid);
+    }
+
+    private static Stream<String> getaVoid(JsonNode topOrg) {
+        var spliterator = Spliterators.spliteratorUnknownSize(topOrg.fieldNames(), Spliterator.ORDERED);
+        return StreamSupport.stream(spliterator, false);
+    }
+
+    private static void assertAllFieldsAreValid(String fieldName) {
```
Comment (sondrev) [inline expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java:269]: The method name does not really describe what the method does. There is no "all" inside the method. Maybe rename to assertFieldIsValid instead?

File: expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java [hunk 2]
```diff
@@ -277,6 +253,41 @@ void shouldNotFailWhenInputContainsAffiliationsThatAreIncomplete() {
...
 
+
+    private void assertExplicitFieldsFromFraming(ObjectNode framedResultNode) {
+        var node = framedResultNode.at("/topLevelOrganization");
+        StreamSupport.stream(node.spliterator(), false)
+            .flatMap(ExpandedResourceTest::getaVoid)
+            .forEach(ExpandedResourceTest::assertAllFieldsAreValid);
+    }
+
+    private static Stream<String> getaVoid(JsonNode topOrg) {
```
Comment (sondrev) [inline expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java:264]: Its hard to udnerstand this method, and the name does not help. I read it as "get a void" but that does not make any sense

**Review by ketilaa — APPROVED** [review]
Nothing to add besides what @sondrev commented on.

File: expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java (hunk 2 above)
Comment (brinxmat) [reply expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java:264]: Auto-refactor, alas. I thought I had renamed it; I was mistaken

**Review by sondrev — APPROVED** [review]

**Pull request merged**